packages/snap/test/rpc.test.ts

```diff
@@ -543,11 +545,13 @@ describe('Test rpc handler function', function () {
         .withArgs({ operation: 'get' })
         .resolves(createState([testHolder], []));
 
-      const unknownZkCert = { ...zkCert };
+      const unknownZkCert = { ...zkCert } as ZkCertRegistered<
+        Record<string, unknown>
+      >;
```

Comment:
It could be expressed better without making a cast.

```typescript
const unknownZkCert: ZkCertRegistered<Record<string, unknown>> = { ...zkCert };
```